tools/workspace/yaml_cpp/repository.bzl

```
@@ -17,6 +17,14 @@ def _impl(repository_ctx):
     elif os_result.is_ubuntu:
         build_flavor = "ubuntu-{}".format(os_result.ubuntu_release)
         repository_ctx.symlink("/usr/include/yaml-cpp", "include/yaml-cpp")
+    elif os_result.is_manylinux:
+        build_flavor = "ubuntu-20.04"
```

Uh... this didn't seem to not work, but shouldn't this be:
```suggestion
        build_flavor = "ubuntu-18.04"
```

---

Added a comment to explain what's happening here.